ui/webui/test/machine_install.py

```diff
@@ -113,3 +113,9 @@ def kill(self):
         )
         if self.http_server:
             self.http_server.kill()
+
+    def add_disk(self, size=None):
```

Comment:
I think this will fail if size is not set ? I think there should either be a default value or the argument should be required, not optional.

Reply:
Good point, will update